src/components/Settings/components/Pages/HistorySetting.jsx

```diff
@@ -35,8 +34,14 @@ const HistorySetting = () => {
 			const history = await getDocs(collection(db, `users/${user.uid}/history/`));
 			history.forEach((doc) => {
 				const data = doc.data();
-        const hours = new Date(data.visitTime).getHours() <= 9 ? '0' + new Date(data.visitTime).getHours() : new Date(data.visitTime).getHours();
-        const minutes = new Date(data.visitTime).getMinutes() <= 9 ? '0' + new Date(data.visitTime).getMinutes() : new Date(data.visitTime).getMinutes();
+				const hours =
+					new Date(data.visitTime).getHours() <= 9
+						? '0' + new Date(data.visitTime).getHours()
+						: new Date(data.visitTime).getHours();
+				const minutes =
+					new Date(data.visitTime).getMinutes() <= 9
+						? '0' + new Date(data.visitTime).getMinutes()
+						: new Date(data.visitTime).getMinutes();
```

Comment:
Je maakt hier verschillende keren `new Date(data.visitTime)` aan. Ik zou dit 1x doen en dat in een `date` variable steken ofzo. Die kan je dan altijd gebruiken